### DISCUSSION TITLE
Is there anyway I can track a conversation in which the AI is context-aware?

### DISCUSSION THREAD

**Discussion by DiamondGo** [sashabaranov/go-openai#148] [General]
I don't see anything like a conversation id, will there be one? Thank you.

**Comment by DiamondGo** [comment]
It looks like I need to put all the old questions into the message queue. Don't know whether it's the correct way to do so.

**Reply by Darwinium** [reply]
This is what I am doing - I am including the conversation history in the request, along with the request itself. Please keep in mind that this will increase the price, as it will require more tokens.

**Comment by ahmedRSA** [comment]
save every prompt and response in db 
and when you need context use that history in prompt to ask questions
